### DISCUSSION TITLE
Other Languages support

### DISCUSSION THREAD

**Discussion by yDeed** [aleganza/akuse#69] [General]
How could I help bring fonts in Brazilian Portuguese to the app?

**Comment by aleganza** [comment]
add a Portuguese anime provider with a pull request to [consumet api](https://github.com/consumet/consumet.ts)
when you add it, the pr gets accepted and it is published, you will have to wait for me to finish the current React port of the app, and then make a pr to Akuse with the new consumet portuguese provider

**Reply by yDeed** [reply]
Oh yeah, okay bro. I just made requests for 3 sources in the repo, as soon as they are ready for consumption I will get back to you. If you need help with the port, I'll see what I can help with too.

**Reply by aleganza** [reply]
I see them. The fact is that it is unlikely that anyone will add them. I'll see what I can do in my free time.
By the way are you a programmer? Could you do it by yoursef?

**Reply by yDeed** [reply]
Im a programmer too, but unfortunately my time is also very limited. But I'll take a look at the code on the weekends until they return the API. Btw, thank you very much for the help
